go.mod

```diff
@@ -2,4 +2,13 @@ module github.com/alecthomas/protoparser
 
 go 1.17
 
-require github.com/alecthomas/participle/v2 v2.0.0-alpha7
+require (
+	github.com/alecthomas/participle/v2 v2.0.0-alpha7
+	github.com/stretchr/testify v1.4.0
```

Comment:
I was hoping to exclude testify in this package, just to keep it super lean?

Reply:
👍  Sounds good, will remove